ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -61,14 +61,41 @@ jobs:
         timeout: 1800
         topology: *build
 
+  pki-fedora/build:
+    requires: []
+    priority: 100
+    job:
+      class: Build
+      args:
+        git_repo: '{git_repo}'
+        git_refspec: '{git_refspec}'
+        template: &pki-master-latest
+          name: freeipa/pki-master-f34
+          version: 0.0.1
+        timeout: 1800
+        topology: *build
+
   fedora-latest/temp_commit:
     requires: [fedora-latest/build]
     priority: 50
     job:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_ipa_cert_fix.py::TestIpaCertFix::test_missing_startup
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *master_1repl
+
+  pki-fedora/temp_commit:
+    requires: [pki-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{pki-fedora/build_url}'
```

Comment:
`update_packages: True` is needed in order to make sure the latest nss version is installed (otherwise server install may fail, see https://pagure.io/freeipa/issue/8935)